src/util/std_code.h

```diff
@@ -143,7 +143,8 @@ class code_blockt:public codet
       }
       else if(statement==ID_label)
       {
-        assert(last->operands().size()==1);
+        //  TODO: consider appropriate invariant message
+        INVARIANT(last->operands().size()==1, "");
```

Comment:
`DATA_INVARIANT(..., "Label statements have exactly one parameter (the code that is labelled)")`